.tekton/chrome-service-pull-request.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: '3'
     pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main"
+    pipelinesascode.tekton.dev/pipeline: https://github.com/RedHatInsights/konflux-pipelines/raw/main/pipelines/platform-ui/docker-build-run-unit-tests.yaml
```

Comment:
@JetyAdam this is what is causing the failures - Go is not installed as part of the UI builds. Can we switch to the non-ui docker build?

Reply:
If the non-ui docker build does not run tests, we may need to update the shared pipelines to have an option to do so